kie-server-parent/kie-server-remote/kie-server-client/src/main/java/org/kie/server/client/impl/AbstractKieServicesClientImpl.java

```diff
@@ -210,6 +210,27 @@ public KieServerHttpRequest doOperation(String url) {
         }
     }
 
+    protected <T> T makeHttpGetRequestAndCreateCustomResponseWithHandleNotFound(String uri, Class<T> resultType) {
+        logger.debug("About to send GET request to '{}'", uri);
+        KieServerHttpRequest request = invoke(uri, new RemoteHttpOperation() {
+            @Override
+            public KieServerHttpRequest doOperation(String url) {
+                return newRequest(url).get();
+            }
+        });
+        KieServerHttpResponse response = request.response();
+
+        owner.setConversationId(response.header(KieServerConstants.KIE_CONVERSATION_ID_TYPE_HEADER));
```

Comment:
@bxf12315 This first part of the method is repeated many times, I guess if we can create a private method with this and call if from makeHttpGetRequestAndCreateCustomResponse, makeHttpGetRequestAndCreateCustomResponseWithHandleNotFound and makeHttpGetRequestAndCreateRawResponse. WDYT?

Reply:
ok